articles/virtual-machines/workloads/hpc/enable-infiniband.md

```diff
@@ -33,20 +33,19 @@ If you're using a custom VM image or a [GPU optimized](../../sizes-gpu.md) VM, y
 
 To manually configure InfiniBand with SR-IOV, use the following steps. The example in these steps shows syntax for RHEL/CentOS, but the steps are general and can be used for any compatible operating system such as Ubuntu (16.04, 18.04 19.04) and SLES (12 SP4 and 15). The inbox drivers work as well, but the Mellanox OpenFabrics drivers provide more features.
 
-For more information on the supported distributions for the Mellanox driver, see the latest [Mellanox OpenFabrics drivers](https://www.mellanox.com/page/products_dyn?product_family=26). For more information on the Mellanox OpenFabrics driver, see the [Mellanox user guide](https://docs.mellanox.com/category/mlnxofedib).
+For more information on the supported distributions for the Mellanox driver, see the latest [Mellanox OpenFabrics drivers](https://www.mellanox.com/page/products_dyn?product_family=26). For more information on the Mellanox OpenFabrics driver, see the [Mellanox user guide](https://docs.mellanox.com/category/mlnxofedib). We have examples for other OS distributions in our [AzureHPCImages repository](https://github.com/Azure/azhpc-images).
 
-See the following example for how to configure InfiniBand on Linux:
+See the following example for how to configure InfiniBand on Ubuntu:
```

Comment:
Generally, this example is applicable to various Linux distros (incl. CentOS/RHEL, Ubuntu). Hence the original statement which can be improved. However the new statement is fine too since this is an example.